frameworks/g-cloud-12/questions/services/vulnerabilityManagementApproach.yml

```diff
@@ -18,8 +18,8 @@ max_length_in_words: 100
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter description.
   - name: under_100_words
-    message: Your answer must be no longer than 100 words.
+    message: Description must be 100 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?